client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/placeholder/index.tsx

```diff
@@ -0,0 +1,32 @@
+import './style.scss';
+
+export const PlaceholderThankYou = () => {
+	return (
+		<div className="congrats-placeholder">
```

Comment:
Is there any advantage to leveraging styling from the subcomponents of `<ThankYouV2>`, so we have similar paddings whatnot between header, products, footer? Or do you think we're better off styling the placeholder page independently?

Reply:
I believe that it's better to style it independently, since it's universal loader and it's better to separate code/styles as much as possible to simplify maintenance in the future. Now it's encapsulated and we can work with it like with `black box`, and it's cool since nobody will break any styles etc unintentionally.